### PR TITLE
How to locate things in the book-keeping

### PR DESCRIPTION
**Pull request by betatim**
Started work on how to locate events in the book-keeping. This is still WIP (Work in progress)

Describing how to drive a GUI in words is surprisingly sub-optimal.

### CODE REVIEW

File: 02-bookkeeping.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+"Bookkeeping tree" window. Navigate to the `MC/2012` folder. This will
+give you a very long list of all possible decay types for which there
+is simulated data. We are looking for a folder which is named
+`27163003 (Dst_D0pi,Kpi=DecProdCut)`. The number is a numerical
+representation of the [event
+type](https://cds.cern.ch/record/855452?ln=en).  The text is the human
+readable version of that.
+
+This sample of simulated events will only contain decays of the type
+D* -> D(-> K pi)pi.
```
Comment (alexpearce) [inline 02-bookkeeping.md]: How pedantic do we want to be here? Technically, each generated event will contain a signal decay (as a set of MC particles) that passes the generator level cuts _and_ the rest of the event. Each event might not contain the signal in the reconstruction.

File: 02-bookkeeping.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+If you expand the `27163003 (Dst_D0pi,Kpi=DecProdCut)` folder you
+will find a list of different configurations depending on the
+magnet polarity (`MagDown` and `MagUp`) as well as different
+versions of `Pythia` used to generate the events. We will use
+`Beam4000GeV-2012-MagDown-Nu2.5-Pythia8`.
+
+Next you need to choose what verion of the simulation you want to
```
Comment (alexpearce) [inline 02-bookkeeping.md]: "version"

File: 02-bookkeeping.md (hunk 1 above)
Comment (betatim) [inline 02-bookkeeping.md]: What I wanted to say is that it only contains signal, not background. Anticipating the need to explain why in simulation you have all these different sub-directories where as in the data there is just "the data".

**Comment by betatim** [comment]
For now I'd go for "not so pedantic", especially on language things, as long as the commands/steps are clear/correct

**Comment by kdungs** [comment]
How about some pictures here?

**Comment by betatim** [comment]
Need to add the example output file we download once the BK resumes working.

evt+std://MC/2012/27163003/Beam4000GeV-2012-MagDown-Nu2.5-Pythia8/Sim08e/Digi13/Trig0x409f0045/Reco14a/Stripping20NoPrescalingFlagged/ALLSTREAMS.DST

**Comment by betatim** [comment]
Added the `.py` file you should obtain from the book-keeping.

Following https://github.com/swcarpentry/lesson-example/blob/gh-pages/LAYOUT.md#code-data-and-figures I decided to place it in `data/`.

I am good for this to be merged, if you also agree, hit the green button please. (following the house rule that you need someone else to do the merge)

**Comment by betatim** [comment]
ping @ibab @alexpearce this can be merged before it ends up in conflict again.

**Pull request merged**